{"level": "error", "logger": "__main__", "timestamp": "2022-06-06T22:42:57.035898Z", "filename": "main.py", "func_name": "do_the_math", "lineno": 99}
